auto_backup/models/db_backup.py

```diff
@@ -252,7 +253,8 @@ def schedule_backup(self):
                     # Close the SFTP session.
                     sftp.close()
                 except Exception as e:
-                    _logger.debug('Exception! We couldn\'t back up to the FTP server..')
+                    sftp.close()
+                    _logger.error('Exception! We couldn\'t back up to the FTP server..')
```

Comment:
Actually, just looking at this. Let's improve the logging as we're modifying the line already.
```suggestion
                    _logger.error('Exception! We couldn\'t back up to the FTP server. Here is what we got back instead: %s' % str(e))
```

Reply:
Good catch, I think I had that in mind also but forgot to add it in. Committed and tested, here's an example of the output:
```
2020-07-06 15:43:52,608 24812 INFO xx odoo.addons.auto_backup.models.db_backup: Checking expired files 
2020-07-06 15:43:52,612 24812 INFO xx paramiko.transport.sftp: [chan 0] sftp session closed. 
2020-07-06 15:43:52,612 24812 ERROR xx odoo.addons.auto_backup.models.db_backup: Exception! We couldn't back up to the FTP server. Here is what we got back instead: 'SFTPClient' object has no attribute 'isfile' 
```